src/sections/MainSection.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+import homeVideo from "../assets/home.mp4";
+import Button from "../components/Button";
+
+const MainSection = () => (
+  <SectionContainer>
+    <video
+      style={styles.video}
+      src={homeVideo}
+      autoPlay={true}
+      loop={true}
+      muted={true}
+    />
+    <OverlayContent>
+      <Title>KANEKI RAMOS</Title>
+      <Details>
+        Full Stack Developer / Game Designer / Software Engineer
+      </Details>
+      <ButtonWrapper>
+        <Button style={styles.Button}>Check me out</Button>
+        <Button style={styles.Button}>Get in contact</Button>
+      </ButtonWrapper>
+    </OverlayContent>
+  </SectionContainer>
+);
+
+const styles = {
+  video: {
+    objectFit: "cover",
+    height: "100vh",
+    width: "100vw",
+  },
+
+  Button: {
```

Comment:
not a fan of this here, you might want to think on the different buttons you have on your project and how to unify them to avoid having to write custom styling for each of them, i feel we can do a better job with the buttons